pyface/ui/wx/data_view/tests/test_data_wrapper.py

```diff
@@ -8,14 +8,18 @@
 #
 # Thanks for using Enthought open source!
 
-from unittest import TestCase
-
-import wx
-
-from pyface.ui.wx.data_view.data_wrapper import DataWrapper
-
-
-class TestDataWrapper(TestCase):
+import unittest
+
+try:
+    import wx
+except ImportError:
+    wx_available = False
+else:
+    from pyface.ui.wx.data_view.data_wrapper import DataWrapper
+    wx_available = True
+
+@unittest.skipUnless(wx_available, "Test requires wx")
```

Comment:
nit - i think we need an extra space to prevent flake8 from complaining.

```suggestion

@unittest.skipUnless(wx_available, "Test requires wx")
```